src/main/scala/org/ergoplatform/explorer/db/dao/OutputsDao.scala

```diff
@@ -30,12 +30,18 @@ class OutputsDao extends JsonMeta {
   def findAllByErgoTree(ergoTree: String): ConnectionIO[List[ExtendedOutput]] =
     OutputsOps.findByErgoTree(ergoTree).to[List]
 
+  def findAllByErgoTreeRoot(ergoTreeRoot: String): ConnectionIO[List[ExtendedOutput]] =
+    OutputsOps.findByErgoTreeRoot(ergoTreeRoot).to[List]
```

Comment:
@greenhat ErgoTreeRoot is confusing name. I suggest to use ErgoTreeTemplate. 
And let's start adding ScalaDoc for all new public methods. 
Also in the ScalaDoc it would be useful to refer to https://github.com/ScorexFoundation/sigmastate-interpreter/issues/264 and ErgoTree class to connect things.

Reply:
@aslesarenko I don't like `ErgoTreeRoot` either but I thought the consistency is more important. We called it [`ErgoTree.root`](http://github.com/ScorexFoundation/sigmastate-interpreter/blob/633efcfd47f2fa4aa240eee2f774cc033cc241a5/sigmastate/src/main/scala/sigmastate/Values.scala#L889-L889) in ErgoTree. I suggest we rename it to `template` as well.

Reply:
@aslesarenko I renamed `ErgoTreeRoot` to `ErgoTreeTemplate` along with other usages of the "root". What about `ErgoTree.root` renaming?

Reply:
Yes, make sense to rename ErgoTree.root as well.